channels/applicationinsights-channel-js/src/Offline.ts

```diff
@@ -1,92 +1,104 @@
-import { getWindow, getDocument, getNavigator, isUndefined, isNullOrUndefined, attachEvent } from "@microsoft/applicationinsights-core-js";
-import dynamicProto from "@microsoft/dynamicproto-js";
+import { getWindow, getDocument, getNavigator, isUndefined, isNullOrUndefined, createUniqueNamespace, mergeEvtNamespace, eventOn, eventOff } from "@microsoft/applicationinsights-core-js";
+
+export interface IOfflineListener {
+    isOnline: () => boolean;
+    isListening: () => boolean;
+    unload: () => void;
+}
+
+function _disableEvents(target: any, evtNamespace: string | string[]) {
+    eventOff(target, null, null, evtNamespace);
```

Comment:
This will remove ALL of the events on the target object that match the provided namespace.
So both the `online` and `offline` will be removed by this single call.